lib/operations/misc.js

```diff
@@ -101,22 +142,91 @@ misc.difference = function (x, y) {
 }
 
 /**
- * Computes the distance of the bounds of an interval
+ * Computes the width of the interval. Also known as diameter. 
  * @param {Interval} x
  * @returns {number}
  */
-misc.width = function (x) {
+misc.wid = function (x) {
```

Comment:
I agree that keeping it width would be more readable and understandable. It just is that in the math. literature a 3 letters handle is preferred (example: mig, mag, abs, etc). 

An alias is something I haven't thought about and perhaps is the golden mean of the two approaches.